go.mod

```diff
@@ -241,6 +241,8 @@ require (
 	github.com/stretchr/objx v0.2.0 // indirect
 )
 
+require github.com/apparentlymart/go-cidr v1.1.0
```

Comment:
I was going to object to the new dependency, but I see that this is identical to what cluster-dns-operator is doing (https://github.com/openshift/cluster-dns-operator/blob/b9c9dfe826b51c4b4c93dc0619178f67842ab385/pkg/operator/controller/controller.go#L560-L568).

Any idea why there are so many separate `require` stanzas here?

Reply:
I think the require structure has to do with the rebase script. @fzdarsky am I remembering that correctly?

Reply:
That's an annoying behaviour by `go mod` since 1.17. It should only have two blocks, one for direct and one for indirect, but it randomly creates new blocks. Haven't figured out the system behind that. Anyway, the rebase script is just yet another victim of this behaviour.